src/main/java/net/sf/rails/game/specific/_1856/CGRFormationRound.java

```diff
@@ -575,7 +572,7 @@ private void formCGR () {
             // First collect old names per city
             Map<String, String> oldTokens = new HashMap<String, String>();
             String cityName;
-            for (BaseToken token : nonHomeTokens) {
+            for (BaseToken token : nonHomeTokens ) {
```

Comment:
Why the empty space here?

Reply:
Typo? Either way, I've started to run the classes through a code formatter so we have same style everywhere but not in a place to commit and push that-- not with other, more important changes happening